pkg/controller/certificates/approver/sarapprove.go

```diff
@@ -192,9 +194,23 @@ var kubeletServerUsages = []capi.KeyUsage{
 }
 
 func isSelfNodeServerCert(csr *capi.CertificateSigningRequest, x509cr *x509.CertificateRequest) bool {
+	if !utilfeature.DefaultFeatureGate.Enabled(features.RotateKubeletServerCertificate) {
```

Comment:
I recommended to @jcbsmpsn that we do this. I really don't want to be died to the behaviour of isSelfNodeServer until we have thought through kubelet's self reporting their SANs. What we currently allow is too weak, and this is an API.